ntoskrnl/ke/bug.c

```diff
@@ -291,6 +346,12 @@ KeRosDumpStackFrames(IN PULONG_PTR Frame OPTIONAL,
     }
 }
 
+/*
+ * @brief brief discription here.
+ *
+ * Full discription here.
+*/
+INIT_FUNCTION
```

Comment:
If you want to write documentation, why do you add code?

Reply:
That was gitlab merging master into my branch.